src/pages/filter/components/timePerMonth-checkboxes/timePerMonth-checkboxes.component.tsx

```diff
@@ -19,7 +19,7 @@ export const TimePerMonthCheckboxes: React.FC<Partial<InputRadioProps>> = props
 
         <InputRadio id="radio-3" name="timePerMonth" lable="Високу" value="high" refAttribute={props.refAttribute} />
 
-        <InputRadio id="radio-4" name="timePerMonth" lable="Будь-яку" value="any" refAttribute={props.refAttribute} />
+        {/* <InputRadio id="radio-4" name="timePerMonth" lable="Будь-яку" value="any" refAttribute={props.refAttribute} /> */}
```

Comment:
No unused code

Reply:
Yes I know, but it is because there is no final decision on these things and we can still come back to them. So I commented  that part of code, and after the financial decision I will delete it in the next pull of requests